lib/matplotlib/offsetbox.py

```diff
@@ -263,32 +263,22 @@ def set_offset(self, xy):
         xy : (float, float) or callable
             The (x, y) coordinates of the offset in display units. These can
             either be given explicitly as a tuple (x, y), or by providing a
-            function that converts the extent into the offset. This function
-            must have the signature::
-
-                def offset(width, height, xdescent, ydescent, renderer) \
--> (float, float)
+            function dynamically computes an offset (taking the arguments
+            passed to `.OffsetBox.get_offset`).  It is recommended, but not
+            compulsory, to make such functions take no arguments.
```

Comment:
Let's not become more permissive. Either no arguments, or `def offset(width, height, xdescent, ydescent, renderer)`.